packages/gitbeaker-core/src/infrastructure/RequestHelper.ts

```diff
@@ -68,14 +68,12 @@ async function get(
     totalPages: parseInt(headers['x-total-pages'], 10),
   };
 
-  const underLimit = maxPages ? pagination.current < maxPages : true;
```

Comment:
woudldnt this completely ignore the maxPages limit?

Reply:
That's true, but there is no way to know which the current page is from there. We could add some counter somewhere maybe? Not sure where to put that though.